modules/validations.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] please avoid disabling any linters, linters are used to improve your code and make it more readable.